etc/worker.yaml

```diff
@@ -58,7 +58,10 @@ store:
   path: "/var/lib/sonm/worker.boltdb"
 
 benchmarks:
+  # URL to download benchmark list, use `file://` schema to load file from a filesystem.
   url: "https://raw.githubusercontent.com/sonm-io/allowed-list/master/benchmarks_list.json"
+  # downloading can be skipped for development purposes
+  # disable: true
```

Comment:
Better not to specify the URL at all instead of this. So make both `url` and  `benchmarks` (for now) optional.

Reply:
Done.